### PR TITLE
fix(text-input-x): add disabled tokens

### PR DESCRIPTION
**Pull request by SimonFinney**
Text input and label color values were displaying incorrectly for the disabled state at 'Gray 100' - Please advise if these token values are incorrect.

#### Changelog

**Changed**

- Update text input disabled state variables to use `$disabled-*` tokens

### CODE REVIEW

File: src/components/form/_form.scss [hunk 1]
```diff
@@ -133,7 +133,7 @@
   }
 
   .#{$prefix}--label--disabled {
-    color: $disabled;
+    color: $disabled-03;
```
Comment (aagonzales) [inline src/components/form/_form.scss]:
```suggestion
    color: $disabled-02;
```

Label should be `$disabled-02` 👍

**Review by aagonzales — COMMENTED** [review]
Just the label needs an update the other ones are correct

**Comment by asudoh** [comment]
Thanks @aagonzales for reviewing!

**Comment by SimonFinney** [comment]
Thanks - this is ready for review again!

**Review by asudoh — APPROVED** [review]
LGTM 👍 - Thanks @SimonFinney!

**Review by aagonzales — APPROVED** [review]
Perfect!

**Pull request merged**